services/ux-backend/main.go

```diff
@@ -94,3 +104,25 @@ func main() {
 	log.Fatal(err)
 
 }
+
+func newClient() (client.Client, error) {
+	klog.Info("Setting up k8s client")
+	scheme := runtime.NewScheme()
+	if err := ocsv1.AddToScheme(scheme); err != nil {
+		return nil, err
+	}
```

Comment:
are we fetching any resources created by ocsv1? do we need to add it to the scheme?

Reply:
client gets failed, if i did not added the ocsv1 schema, the same is followed in onboarding-validation-key-generator main.go

Reply:
the onboarding-validation-key-generator requires us to access the storageCluster hence the ocsv1 scheme is added to the client we create there. The ux-backend for now only fetches the secret. I don't think we need the ocsv1 scheme here

Reply:
ack, removed the ocsv1 schema